extensions/wikidata/tests/src/org/openrefine/wikidata/qa/MockConstraintFetcher.java

```diff
@@ -59,6 +60,11 @@
     
     public static PropertyIdValue propertyOnlyPid = Datamodel.makeWikidataPropertyIdValue("P372");
 
+    public static PropertyIdValue conflictsWithPid = Datamodel.makeWikidataPropertyIdValue("P50");
+    public static PropertyIdValue pidConflictingStatement = Datamodel.makeWikidataPropertyIdValue("P31");
+    public static Value conflictingStatementValue = Datamodel.makeWikidataItemIdValue("P31");
```

Comment:
`Datamodel.makeWikidataItemIdValue("P31")` is incorrect, you are trying to create an `ItemIdValue` with a property identifier.